README.md

```diff
@@ -0,0 +1,28 @@
+<h2>dev-ace: An AI Engine Based on LLaMA</h1>
+
+<p>
+    This repository, is a branch of the 
+    <a href="https://github.com/conormckenzie/reasoning-engine">reasoning-engine</a> project, 
+    focusing on developing an AI engine based on LLaMA.
+</p>
+
+<h2>Overview</h2>
+<p>
+    The branch will be used to push and pull to and from the main branch
+</p>
+
+<h2>Features</h2>
+<ul>
+
+</ul>
+
+<h2>Acknowledgments</h2>
+<ul>
+    <li><a href="https://github.com/conormckenzie/reasoning-engine">Original reasoning-engine repository</a></li>
+    <li>The LLaMA model by Meta AI</li>
```

Comment:
The project doesn't have anywhere that it uses Llama yet, so this can be dropped for now.